algorithms/src/std_algorithms/impl/Kokkos_IsPartitioned.hpp

```diff
@@ -107,14 +107,27 @@ bool is_partitioned_impl(const std::string& label, const ExecutionSpace& ex,
   constexpr index_type red_id_max =
       ::Kokkos::reduction_identity<index_type>::max();
 
+  // clang-format off
```

Comment:
Why disable the format here?

Reply:
readability (IMO)

Reply:
i will revert

Reply:
You can post the diff to try to convince me or see if others will agree with you.
I will not block but would only apply at this time if the format disable is dropped.

Reply:
I am not trying to be difficult. I just really cannot tell what was better with disabling the format.

Reply:
To me it was easier to glance and see the various ifs/elses without having like 10 lines all condensed but was just a personal preference I guess :) 

Really no big deal to revert it back